framework/src/application/schema/application_config_schema.js

```diff
@@ -17,357 +17,299 @@
 module.exports = {
 	id: '#/app/config',
 	type: 'object',
-	required: ['app', 'components', 'modules'],
+	required: [
+		'version',
+		'protocolVersion',
+		'ipc',
+		'genesisConfig',
+		'forging',
+		'network',
+		'components',
+		'modules',
+	],
 	properties: {
-		app: {
+		label: {
+			type: 'string',
+			pattern: '^[a-zA-Z][0-9a-zA-Z\\_\\-]*$',
+			minLength: 1,
+			maxLength: 30,
+			description:
+				'Restricted length due to unix domain socket path length limitations.',
+		},
+		version: {
+			type: 'string',
+			format: 'version',
+		},
+		protocolVersion: {
+			type: 'string',
+			format: 'protocolVersion',
+		},
+		buildVersion: {
+			type: 'string',
+			example: 'v09:54:35 12/04/2017',
```

Comment:
at the moment the value is `2020-01-16T13:43:35.000Z` did it change?

Reply:
seems like `example: 'v09:54:35 12/04/2017',` was the original example too, but im not sure what's the format now.

Reply:
i referred from here https://node.lisk.io/api/node/constants, @fchavant any thoughts?

Reply:
The format has been `%Y-%m-%dT%H:%M:%S.000Z` for as long as I can remember. Note that unlike `v09:54:35 12/04/2017` it can be easily sorted.

Reply:
Addressed